src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -4,7 +4,7 @@
         <div id="headers" class="row font-weight-bold pt-2">
             <div class="col-md-1 header-cell"></div>
             <div class="col-md-3 header-cell" v-translate="'projectName'"></div>
-            <div class="col-md-1 header-cell">Versions</div>
+            <div class="col-md-1 header-cell" v-translate="'versions'"></div>
```

Comment:
We didn't need to translate this for French, since version(s) is the same in French as in English...